src/routes/modifyGame.ts

```diff
@@ -92,4 +59,46 @@ router.get(
   }
 )
 
+async function getGameIdIfNotStarted(
```

Comment:
Maybe 'SELECT game_id, current_player....' ? That way we do not return null, seems like a nicer way

Reply:
And then this function becomes more general, as it returns also started games

Reply:
I believe that it is better to have two function. Note that we never even use current_player and we would have to do additional if in main function just to check if the game has not started.

Reply:
We still do if in main (modifyGame, line 39) to check if we returned null, so there would no new if to the logic AFAIK. It just seems better to avoid using NULLs to mark abscence of the value; it is better to check for it explicitly via, for example current_player = null (yea we still use nulls but on db level - can't get rid of that). Then the function does more - it returns id of started game, as well of those that are not started (which will be used in PR regarding game logic, I think), and does not become harder to use for the present use case (just more verbose)

Reply:
Let's maybe try something completely different then. I think just returning the `gameId` is not enough, as we would then follow up with another query to check if it has started - I think this is just redundant code.

So why don't we add a field into the returned object `started: Booolean`? We could then always return the gameId, while both avoiding the usage of nulls and having to do another query.

What do you think about it @sormys ?